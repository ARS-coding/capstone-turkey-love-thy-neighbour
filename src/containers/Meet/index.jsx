/* eslint-disable prettier/prettier */
import React from "react";

import { Container, Row, Col } from "react-bootstrap";

import { CTAButton } from "../../components/CustomButtons/index";
import MeetImg from "../../images/meet.png";
import "./index.scss";
// use https://lipis.github.io/bootstrap-sweetalert/ the second from the bottom for the how was your meeting alert

function Meet() {
  return (
    <Container fluid className="meet-container-fluid">
      <Container className="meet-container d-flex align-items-center">
        <Row className="meet-row">
          <Col>
            <h3>Hello, Ali</h3>
            <h1>Meet nearby people with our cutting edge algorithms.</h1>
            <p>
              After you have been matched with people near you, you can send
              them a message and meet up quickly !
            </p>
            <CTAButton>Meet Now</CTAButton>
          </Col>
          <Col>
            <img src={MeetImg} className="img-fluid" alt="meet img" />
          </Col>
        </Row>
      </Container>
    </Container>
  );
}

export default Meet;
