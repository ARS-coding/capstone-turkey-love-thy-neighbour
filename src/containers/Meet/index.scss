@import "../../style/_variables.scss";

.meet-container-fluid {
  height: calc(100vh - 60px);

  & > .meet-container {
    height: 100%;

    & > .meet-row {
      margin: 0;

      & > div:first-child {
        padding: 0;

        h1 {
          color: $logo-red;

          @media screen and (max-width: 1399px) {
            font-size: 38px;
          }
          @media screen and (max-width: 1199px) {
            font-size: 28px;
          }
          @media screen and (max-width: 991px) {
            font-size: 23px;
          }
        }

        h3 {
          color: $logo-blue;

          @media screen and (max-width: 1199px) {
            font-size: 25px;
          }
          @media screen and (max-width: 991px) {
            font-size: 19px;
          }
        }

        p {
          color: $black;

          @media screen and (max-width: 991px) {
            font-size: 13px;
          }
        }
      }
      & > div:last-child {
        padding: 0;

        img {
          width: 500px;
        }
      }
    }
  }
}
