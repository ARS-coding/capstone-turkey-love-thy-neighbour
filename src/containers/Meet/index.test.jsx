import React from "react";
import renderer from "react-test-renderer";
import configureStore from "redux-mock-store";
import { Provider } from "react-redux";
import { BrowserRouter as Router } from "react-router-dom";
import Meet from ".";

describe("Meet page", () => {
  const mockState = {
    user: {
      firestoreDoc: {
        firstName: "Test User",
      },
    },
  };
  const mockStore = configureStore();
  let store;
  let tree;

  beforeEach(() => {
    store = mockStore(mockState);
    tree = renderer
      .create(
        <Provider store={store}>
          <Router>
            <Meet />
          </Router>
        </Provider>
      )
      .toJSON();
  });

  it("renders Meet page correctly", () => {
    expect(tree).toMatchSnapshot();
  });
});
