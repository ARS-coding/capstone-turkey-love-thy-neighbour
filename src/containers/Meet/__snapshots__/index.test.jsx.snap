// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Meet page renders Meet page correctly 1`] = `
<div
  className="meet-container-fluid d-flex align-items-center container-fluid"
>
  <div
    className="meet-content-container d-flex align-items-start align-items-sm-start align-items-md-center container"
  >
    <div
      className="meet-row row"
    >
      <div
        className="d-flex flex-column justify-content-center align-items-center align-items-sm-center align-items-md-start text-center text-sm-center text-md-start mb-3 mb-sm-3 mb-md-0 col-md-6 col-sm-12 col-12 order-md-first order-sm-last order-last"
      >
        <h3>
          Hello, 
          Test User
        </h3>
        <h1>
          Meet nearby people with our cutting edge algorithms.
        </h1>
        <p>
          After you have been matched with people near you, you can send them a message and meet up quickly !
        </p>
        <button
          className="sc-bdnxRM sc-gtsrHT hPJaof bLMOay"
        >
          <a
            href="/neighbors"
            onClick={[Function]}
          >
            Meet Now
          </a>
        </button>
      </div>
      <div
        className="d-flex justify-content-end col-md-6 col-sm-12 col-12 order-md-last order-sm-first order-first"
      >
        <img
          alt="meet img"
          className="img-fluid"
          src="meet.jpg"
        />
      </div>
    </div>
  </div>
</div>
`;
